src/main/kotlin/org/opensearch/replication/task/index/IndexReplicationTask.kt

```diff
@@ -399,9 +399,7 @@ open class IndexReplicationTask(id: Long, type: String, action: String, descript
                             continue
                         }
                         val setting = indexScopedSettings[key]
-                        if (setting.isPrivateIndex) {
-                            log.info("private setting $key skipping")
-                        } else {
+                        if (!setting.isPrivateIndex) {
```

Comment:
Wouldn't the visibility help?

Reply:
This doesn't add much value and was left over log by oversight.